avocado/core/remoter.py

```diff
@@ -46,6 +48,25 @@ class ConnectionError(RemoterError):
     pass
 
 
+def _get_env_vars(env_vars):
+    """
+    Gets environment variables.
+
+    :param variables: A list of variables to get.
+    :return: A dictionary with variables names and values.
+    """
+    LOG.info(env_vars)
```

Comment:
Maybe prefix the message with a description of what's the logged values are? Simple stuff as "Environment variables: %s" would make the actual logged message more clear.